commerce-notification-service/src/main/java/com/liferay/commerce/notification/internal/util/CommerceNotificationHelperImpl.java

```diff
@@ -182,99 +158,48 @@ protected void sendNotification(
-			commerceAccountUserIds);
-
-		for (long userId : commerceAccountUserIdsArray) {
-			sendNotification(
-				commerceNotificationTemplate, commerceNotificationType, object,
-				userId);
-		}
-	}
-
-	protected void sendNotificationsToUserIds(
-			long[] userIds, long[] commerceAccountGroupIds,
-			CommerceNotificationTemplate commerceNotificationTemplate,
-			CommerceNotificationType commerceNotificationType, Object object)
-		throws PortalException {
-
-		for (long userId : userIds) {
-			List<CommerceAccountUserRel> commerceAccountUserRelsByUserId =
-				_commerceAccountUserRelLocalService.
-					getCommerceAccountUserRelsByCommerceAccountUserId(userId);
-
-			List<Long> commerceAccountIds = new ArrayList<>();
-
-			for (CommerceAccountUserRel commerceAccountUserRel :
-					commerceAccountUserRelsByUserId) {
-
-				commerceAccountIds.add(
-					commerceAccountUserRel.getCommerceAccountId());
-			}
-
-			long[] commerceAccountIdsArray = ArrayUtil.toLongArray(
-				commerceAccountIds);
-
-			if (ArrayUtil.containsAll(
-					commerceAccountIdsArray, commerceAccountGroupIds)) {
-
-				sendNotification(
-					commerceNotificationTemplate, commerceNotificationType,
-					object, userId);
-			}
+		String to = formatString(
+			commerceNotificationType, commerceNotificationTemplate.getTo(),
+			object, userLocale);
+
+		String[] toUserIds = StringUtil.split(to, ",");
+
+		for (String toUserId : toUserIds) {
+			User toUser = _userLocalService.getUser(
+				GetterUtil.getLong(toUserId));
+
+			_commerceNotificationQueueEntryLocalService.
+				addCommerceNotificationQueueEntry(
+					toUser.getUserId(), groupId,
+					commerceNotificationType.getClassName(object),
+					commerceNotificationType.getClassPK(object),
+					commerceNotificationTemplate.
+						getCommerceNotificationTemplateId(),
+					commerceNotificationTemplate.getFrom(), fromName,
+					toUser.getEmailAddress(), toUser.getFullName(),
+					commerceNotificationTemplate.getCc(),
+					commerceNotificationTemplate.getBcc(), subject, body, 0);
 		}
 	}
 
 	private static final Pattern _placeholderPattern = Pattern.compile(
 		"\\[%[^\\[%]+%\\]", Pattern.CASE_INSENSITIVE);
 
-	@Reference
-	private CommerceAccountGroupLocalService _commerceAccountGroupLocalService;
-
-	@Reference
-	private CommerceAccountUserRelLocalService
-		_commerceAccountUserRelLocalService;
-
 	@Reference
 	private CommerceNotificationQueueEntryLocalService
 		_commerceNotificationQueueEntryLocalService;
 
-	@Reference
-	private CommerceNotificationTemplateCommerceAccountGroupRelLocalService
-		_commerceNotificationTemplateCommerceAccountGroupRelLocalService;
-
 	@Reference
 	private CommerceNotificationTemplateLocalService
 		_commerceNotificationTemplateLocalService;
 
 	@Reference
 	private CommerceNotificationTypeRegistry _commerceNotificationTypeRegistry;
 
+	@Reference
+	private CommerceDefinitionTermContributorRegistry
+		_commerceOrderDefinitionTermContributorRegistry;
```

Comment:
I think this variable should be named `_commerceDefinitionTermContributorRegistry` now (no `Order` in there), unless I'm missing something.